pkg/skaffold/jib/jib_unix.go

```diff
@@ -0,0 +1,40 @@
+// +build linux darwin
+
+/*
+Copyright 2018 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package jib
+
+func getWrapper(defaultExecutable string) string {
+	switch defaultExecutable {
```

Comment:
this smells a little bit like we are trying to decide between maven/gradle logic using arbitrary consts (lack of enums I guess?), while in other cases we are using "Maven/Gradle" postfix. It's starting to get confusing what logic belongs to maven/gradle and what's common. I'd like to see if introducing the subpackages under jib/maven, jib/gradle would help or not.